### PR TITLE
Quantile highlights

### PR DESCRIPTION
**Pull request by IceFreez3r**
Additional button in the market overview before the favorite filter button. On click every item gets highlighted based on the quantile of the price in the latest API call. High price = quantile close to 1 = red. Low price = quantile close to 0 = green. Continuous colors, NO categories.

### CODE REVIEW

File: modules/marketplace.js [hunk 1]
```diff
@@ -410,19 +400,13 @@ body .marketplace-table-cell-div {
...
-                return;
-            } else {
+        if (this.createMap) {
+            let items = document.getElementsByClassName('marketplace-content')[0]; // Overview page
+            if (items) {
                 items = items.firstChild;
-                if (this.createMap) {
-                    this.iconToIdMap(items);
-                }
+                this.iconToIdMap(items);
```
Comment (IceFreez3r) [inline modules/marketplace.js:407]: I think I removed this by mistake

Comment (IceFreez3r) [reply modules/marketplace.js:407]: nvm

File: storage.js [hunk 2]
```diff
@@ -45,11 +47,16 @@ function handleApiData(data) {
             itemList[data[i].itemID]["prices"] = [];
         }
         itemList[data[i].itemID]["prices"].push([timestamp, data[i].minPrice]);
+        sortPriceList(data[i].itemID);
+        itemList[data[i].itemID]["latestPrice"] = data[i].minPrice;
     }
-    itemList[2]["prices"].push([timestamp, heatValue(timestamp).heatValue]);
+    const currentHeatValue = heatValue(timestamp);
```
Comment (IceFreez3r) [inline storage.js:53]: Remove `timestamp` here

File: modules/highlight.js [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+}
+
+.favorite-highlight.heat-highlight {
+    border: 3px solid white;
+    box-shadow: 0 0 0 3px red;
+}
+
+.quantile-borders .favorite-highlight.heat-highlight {
+    box-shadow: 0 0 0 3px white, 0 0 0 6px red;
+}
```
Comment (IceFreez3r) [inline modules/highlight.js:44]: Sometimes spaces in between sometimes not

**Comment by IceFreez3r** [comment]
Didn't check where, but somewhere is a leftover debug console log for the slider

File: modules/highlight.js (hunk 3 above)
Comment (IceFreez3r) [reply modules/highlight.js:44]: Was correct, but I changed it such that all classes are directly set on the item node.

**Review by Z3krom991 — APPROVED** [review]

**Pull request merged**